upi/openstack/security-groups.yaml

```diff
@@ -63,13 +63,55 @@
       port_range_min: 53
       port_range_max: 53
 
+  - name: 'Create master-sg rule "Ingress HTTP"'
```

Comment:
These rules, for master should be conditional to the fact that masters are schedulable. Discussion is happening in https://github.com/openshift/installer/pull/7576/.

Let's first resolve the compact cluster case in in https://github.com/openshift/installer/pull/7576/, before adding the rules for IPv6.
@MaysaMacedo, would that work for you?